api/Document.json

```diff
@@ -7660,7 +7661,8 @@
               "version_added": "56"
             },
             "ie": {
-              "version_added": "10"
+              "version_added": false,
+              "notes": "This event handler attribute is not supported, however the event itself is supported since IE 10. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```

Comment:
```suggestion
              "notes": "This event handler attribute is not supported; however, the event itself is supported since IE 10. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```